README.rst

```diff
@@ -27,11 +27,11 @@ Dependencies
 
 Requirements:
 
-- ``ansible`` (`>= 2.9.5`), ``yq``, ``jq``
+- ``ansible`` (`>= 2.9.5`), ``yq``, ``jq``, ``fire``
 
 .. code-block:: shell
 
-    pip3 install yq ansible==2.9.*
+    pip3 install yq ansible==2.9.* fire
```

Comment:
Here we can change to `pip3 install -r requirements.txt`

Reply:
good idea!